src/GreenPlayer.js

```diff
@@ -23,19 +36,27 @@ class SoundPlayer extends EventEmitter {
 
     /**
      * Handle any event we have told the output node to listen for.
+     * @param {object} event - event to handle
```

Comment:
Think it might be worth mentioning that this is the EventListener DOM interface? https://developer.mozilla.org/en-US/docs/Web/API/EventListener